app/views/troubleshooting/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2 class="heading"><%= t(".page_title.#{@item_type}") %></h2>
+<p><%= t(".page_description.#{@item_type}") %></p>
+
+<% if @item.is_a?(Tag) %>
+  <p><%= link_to @item.name, tag_path(@item) %> (<%= @item.type %>)</p>
+<% elsif @item.is_a?(Work) %>
+  <ol class="work index group">
+    <%= render "works/work_blurb", work: @item %>
+  </ol>
+<% end %>
+
+<%= form_tag troubleshooting_path, method: "patch" do %>
```

Comment:
A question from Support, whose committee will be using these pages: Would it cause problems if someone selected all the checkboxes on this page at once?

Reply:
It should be fine. Some of the checkboxes have overlapping functionality (e.g. fixing filters always triggers a reindex), but it shouldn't cause problems aside from generating extra work for the servers.